gruntwork-install

```diff
@@ -50,26 +50,27 @@ function print_usage {
 
 function log {
   local readonly level="$1"
-  local readonly message="$2"
+  shift
+  local -ra message=("$@")
   local readonly timestamp=$(date +"%Y-%m-%d %H:%M:%S")
   local readonly scriptname="$(basename "$0")"
 
-  >&2 echo -e "${timestamp} [${level}] [$scriptname] ${message}"
+  >&2 echo -e "${timestamp} [${level}] [$scriptname] ${message[@]}"
 }
 
 function log_info {
-  local readonly message="$1"
-  log "INFO" "$message"
+  local -ra message=("$@")
+  log "INFO" "${message[@]}"
 }
 
 function log_warn {
-  local readonly message="$1"
-  log "WARN" "$message"
+  local -ra message=("$@")
+  log "WARN" "${message[@]}"
 }
 
 function log_error {
-  local readonly message="$1"
-  log "ERROR" "$message"
+  local -ra message=("$@")
+  log "ERROR" "${message[@]}"
```

Comment:
I don't know if this is strictly necessary, but the logging was swallowing all but the first parameter when printing the command that was getting executed, so I figured this couldn't hurt.

Reply:
Thanks!